### ISSUE TITLE
Check for git version 2.9+ when installing seekrets

### ISSUE THREAD

**Issue by LinuxBozo** [18F/laptop#79]
In order to make sure sure that seekrets works as intended, the user should have git 2.9+ installed. git 2.9 added a feature that allows setting a global directory to find hooks. This allows seekrets to work without having to find/modify any existing repositories.

Implementation:
`seekrets-install` should check for proper version, and ask/allow user to upgrade with homebrew if version is too old.

If fixing, please use `seekret` branch so it's merged into PR #69

**Comment by LinuxBozo** [comment]
GitHub Desktop git version can be checked, depending on where it's installed like so:
```
~/Applications/GitHub\ Desktop.app/Contents/Resources/git/bin/git --version
```
or 
```
/Applications/GitHub\ Desktop.app/Contents/Resources/git/bin/git --version
```

**Comment by monfresh** [comment]
The laptop script installs the latest version of git via Homebrew, and also automatically updates any outdated tools, so I don't think this check is necessary.

**Comment by LinuxBozo** [comment]
@monfresh Since we're going to allow seekrets to install itself separately as well, to cover those folks who may not run the laptop script in it's entirety, it's a safety net check.

**Comment by monfresh** [comment]
Gotcha.

**Comment by LinuxBozo** [comment]
Will be closed by 923a113
